image/roxctl/rhtap.Dockerfile

```diff
@@ -0,0 +1,53 @@
+# TODO: can we follow tags?
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_1.20 as builder
+
+WORKDIR /go/src/github.com/stackrox/rox/app
+
+# TODO: do we want to avoid this by COPYing each required directory separately?
+COPY . .
+
+RUN git config --global --add safe.directory /go/src/github.com/stackrox/rox/app && \
+    # We could add tag fetching to: https://github.com/redhat-appstudio/build-definitions/blob/main/task/git-clone/0.1/git-clone.yaml
+    # TODO: if git fetch fails, check if privileges/file owners are OK, usually only on local build because repo might have moved further.
+    # TODO: this prevents hermetic builds
+    git fetch --tags --force && \
+    mkdir -p image/bin
+
+ENV CI=1 GOFLAGS="" GOTAGS="release"
+
+RUN RACE=0 CGO_ENABLED=1 GOOS=linux GOARCH=$(go env GOARCH) BUILD_TAG=$(make tag) scripts/go-build.sh ./roxctl && \
+    cp bin/linux_$(go env GOARCH)/roxctl image/bin/roxctl
+
+# TODO: can we follow tags?
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest as app
+
+COPY --from=builder /go/src/github.com/stackrox/rox/app/image/bin/roxctl /usr/bin/roxctl
+
+# TODO: this prevents hermetic builds
```

Comment:
Let's put here what we learned.
```suggestion
# TODO: use hermetic builds when installing/updating RPMs becomes hermetic.
# See https://issues.redhat.com/browse/STONEBLD-704
```

When this remains the only non-hermetic thing, let's comment-out `microdnf upgrade -y --nobest` line. I think, for now `git fetch --tags` is not hermetic too and we need to get rid of that.

Reply:
> When this remains the only non-hermetic thing, let's comment-out microdnf upgrade -y --nobest line. I think, for now git fetch --tags is not hermetic too and we need to get rid of that.

Yes, until https://github.com/redhat-appstudio/build-definitions/pull/615 is merged, both prevent hermetic builds.